FirebaseAppCheck/Tests/Unit/Core/FIRAppCheckTests.m

```diff
@@ -1005,6 +1027,23 @@ - (NSArray *)expectTokenRequestFromAppCheckProvider {
   return @[ expectedToken, storeTokenPromise ];
 }
 
+- (NSArray *)expectLimitedUseTokenRequestFromAppCheckProvider {
+  // 1. Don't expect token to be requested from storage.
+  OCMReject([self.mockStorage getToken]);
+
+  // 2. Expect token requested from app check provider.
+  FIRAppCheckToken *expectedToken = [self validToken];
+  id completionArg = [OCMArg invokeBlockWithArgs:expectedToken, [NSNull null], nil];
+  OCMExpect([self.mockAppCheckProvider getTokenWithCompletion:completionArg]);
+
+  // 3. Don't expect new token to be stored.
```

Comment:
For my own understanding, why don't we expect new token to be stored? And how is it not stored in practice?